### DISCUSSION TITLE
Name squatting policy

### DISCUSSION THREAD

**Discussion by Aleksandras-Novikovas** [community/community#18486] [Other Feature Feedback, Questions, & Ideas]
Does GitHub still enforces name squatting policy as written in [Name Squatting Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy) ?
If yes - how can one report unused names?
If no - why is it not removed from docs?

**Comment by number0x0001** [comment]
Pretty sure you have to contact their support about: https://support.github.com/contact.

**Comment by queenofcorgis** [comment]
Hello 👋🏼 

Confirming that our policy is up to date, you can report names through the support link, but we're no longer accepting requests for dormant user names. Best of luck 😄

**Reply by Aleksandras-Novikovas** [reply]
Could you please specify exact link?

While trying to report abuse - after selecting:
I want to report abusive content or behavior. -> I want to report false or misleading information, impersonation or username squatting. -> Someone is using a username I want.
Button 'Send request' is greyed out.

**Reply by landsman** [reply]
wtf
